stats.py

```diff
@@ -285,55 +267,28 @@ def get_fsrs_stats(self: CollectionStats):
         lim = " AND did IN %s" % lim
 
     (
-        retention,
-        stability,
         card_cnt,
         estimated_total_knowledge,
-        estimated_total_knowledge_notes,
-        note_cnt,
         time_sum,
     ) = retention_stability(lim)
     i = []
-    _line_now(i, "Average predicted retention", f"{retention * 100: .2f}%")
-    _line_now(i, "Average stability", f"{round(stability)} days")
-    i.append(
-        "<tr><td align=left style='padding: 5px'><b>Retention by Cards:</b></td></tr>"
-    )
     _line_now(i, "Total Count", f"{card_cnt} cards")
-    _line_now(
-        i,
-        "Estimated total knowledge",
-        f"{estimated_total_knowledge} cards ({retention * 100:.2f}%)",
-    )
     _line_now(i, "Total Time", f"{time_sum/3600:.1f} hours")
     if time_sum > 0:
         _line_now(
             i,
             "Knowledge acquisition rate",
             f"{estimated_total_knowledge / (time_sum/3600):.1f} cards/hour",
         )
-    i.append(
-        "<tr><td align=left style='padding: 5px'><b>Retention by Notes:</b></td></tr>"
-    )
-    _line_now(i, "Total Count", f"{note_cnt} notes")
-    _line_now(
-        i,
-        "Estimated total knowledge",
-        f"{round(estimated_total_knowledge_notes)} notes ({(estimated_total_knowledge_notes / max(note_cnt, 1)) * 100:.2f}%)",
-    )
     title = CollectionStats._title(
         self,
         "FSRS Stats",
         "Only calculated for cards with FSRS memory states",
     )
     stats_data = _lineTbl_now(i)
     interpretation = (
-        "<p>Note: Unless you have a huge backlog, the average predicted retention will be higher than your desired retention. For details, read the interpretation section.</p>"
-        + "<details><summary>Interpretation</summary><ul>"
-        + "<li><b>Average predicted retention</b>: the average probability of recalling a card today. Desired retention is the retention when the card is due. Average retention is the current retention of all cards, including those that are not yet due. These two values are different because most cards are not usually due. <b>The average predicted retention is calculated using FSRS formulas and depends on your parameters.</b> True retention is a measured value, not an algorithmic prediction. So, it doesn't change after changing the FSRS parameters.</li>"
-        + "<li><b>Stability</b>: the time required for the retention to fall from 100% to 90%.</li>"
+        "<details><summary>Interpretation</summary><ul>"
         + "<li><b>Count</b>: the number of cards with FSRS memory states, excluding cards in the (re)learning stage.</li> "
```

Comment:
> , excluding cards in the (re)learning stage

Is this correct?

Reply:
Seems incorrect. I have forgotten why it said that.

Reply:
I think that it's a relic from the pre-Anki 23.10 days. Probably, (re)learning cards didn't have stability then.